File: moderate/Number-Of-Ones/Number Of Ones.java
import java.io.*;
public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            Show1s(line);
        }
    }
    
    static void Show1s(String line){
        String bin = Integer.toString(Integer.valueOf(line),2);
        System.out.println(bin.length() - bin.replace("1","").length());
    }
}

File: moderate/Number-Of-Ones/Number Of Ones.cs
using System;
using System.IO;
using System.Collections.Generic;

class Program
{
    static void Main(string[] args)
    {
        using (StreamReader reader = File.OpenText(args[0]))
        while (!reader.EndOfStream)
        {
            string line = reader.ReadLine();
            if (null == line)
                continue;
            Show1s(line);
        }
    }
    
    static void Show1s(string line){
        string bin = Convert.ToString(Int32.Parse(line),2).ToString();
        string[] str = bin.Split('1');
        Console.WriteLine(str.Length-1);
    }
}
